website/server/controllers/api-v3/chat.js

```diff
@@ -480,4 +480,58 @@ api.deleteChat = {
   },
 };
 
+/**
+ * @api {post} /api/v3/groups/:groupId/chat/:chatId/approve Approve a group chat message
+ * @apiName ApproveChat
+ * @apiGroup Chat
+ * @apiDescription Approvates a chat message from a group
+ *
+ * @apiParam (Path) {UUID} groupId The group _id ('party' for the user party and 'habitrpg' for tavern are accepted)
+ * @apiParam (Path) {UUID} chatId The chat message _id
+ *
+ * @apiSuccess {Object} data The approved <a href='https://github.com/HabitRPG/habitica/blob/develop/website/server/models/group.js#L51' target='_blank'>chat message</a>
```

Comment:
Should that URL be replaced with https://github.com/HabitRPG/habitica/blob/develop/website/server/models/chat.js now?